tests/connection_utils.py

```diff
@@ -30,6 +30,7 @@ async def disconnect_all_and_reconnect(server: ChiaServer, reconnect_to: ChiaSer
 async def add_dummy_connection(
     server: ChiaServer, self_hostname: str, dummy_port: int, type: NodeType = NodeType.FULL_NODE
 ) -> Tuple[asyncio.Queue, bytes32]:
+    dummy_port = uint16(dummy_port)
```

Comment:
Because I didn't want to annotate another 20-odd calls to this test function (mostly from test_full_node.py)
Agree that uint16 in the param type is better.

Reply:
Since this is only called in test code, it can't pollute the main codebase, either.